pkgs/development/python-modules/tensorflow/default.nix

```
@@ -135,18 +135,17 @@ let
       # libs taken from system through the TF_SYS_LIBS mechanism
       grpc
       sqlite
-      openssl
+      boringssl
       jsoncpp
-      pkgs.protobuf
       curl
+      pybind11
       snappy
-      flatbuffers
+      pkgs.flatbuffers
```

```suggestion
      flatbuffers
```

Pleas inherit flatbuffers in top-level.

---

Note that we need both "core" and "python" flatbuffers so we cannot just inherit "flatbuffers". Changed the names to "flatbuffers-core" and "flatbuffers-python", LMK if you prefer it done differently.